content/applications/finance/accounting/bank/transactions.rst

```diff
@@ -27,8 +27,13 @@ Odoo supports multiple file formats to import transactions:
 - Quicken Interchange Format (.QIF);
 - Belgium: Coded Statement of Account (.CODA).
 
-To import a file, go to the :guilabel:`Accounting Dashboard`, and in the :guilabel:`Bank` journal,
-click on :guilabel:`Import Transactions`.
+To import a file, go to the **Accounting Dashboard**, and in the :guilabel:`Bank` journal,
+click on :guilabel:`Import File`.
+
+.. tip::
+   You can also click :guilabel:`⋮` on the :guilabel:`Bank journal` when on the **Accounting
```

Comment:
Here, "Bank journal" is the guilabel, but above it's "Bank" journal, where only "bank" is the guilabel. Unless there's a specific reason for them to be different, I'd make them both the same.